WordPress/WordPressUITests/Screens/MySiteScreen.swift

```diff
@@ -54,6 +62,20 @@ class MySiteScreen: BaseScreen {
         }
     }
 
+    func gotoPostsScreen() -> PostsScreen {
+        postsButton.tap()
+        return PostsScreen()
+    }
+
+    func gotoMediaScreen() {
+        mediaButton.tap()
+    }
```

Comment:
After running the full screenshot generation script, I noticed that this isn't sufficient to ensure the Media Library is opened. In at least one run it mis-clicked and opened the Comments screen instead. Could we add a Media page object that checks to make sure the correct screen is loaded, and return it here?

Reply:
Uhh....that shouldn't be possible? 

But I added a change in 32bc9e79ed5f45a4233125cad25be7c0788870a6 that'll cause the screenshot generation routine to exit if we hit that...somehow? WTF XCUITest...